src/sdk.js

```diff
@@ -99,21 +101,35 @@ class OpenForm {
 
     this.formObject = formObject;
 
-    // render the wrapping React component
-    ReactDOM.render(
-      <React.StrictMode>
-        <IntlProvider messages={messages} locale={lang} defaultLocale="nl">
-          <ConfigContext.Provider value={{baseUrl: this.baseUrl}}>
-            <FormioTranslations.Provider value={{i18n: translations, language: lang}}>
-              <Router basename={this.basePath}>
-                <App form={this.formObject} />
-              </Router>
-            </FormioTranslations.Provider>
-          </ConfigContext.Provider>
-        </IntlProvider>
-      </React.StrictMode>,
-      this.targetNode,
-    );
+    let doStart = !!new URLSearchParams(window.location.search).get(START_FORM_QUERY_PARAM)
+
+    // Automatically redirect the user to a specific login option (if configured)
+    if(!doStart && formObject.autoLoginAuthenticationBackend) {
+      let loginUrl;
+      formObject.loginOptions.forEach(option => {
+        if(option.identifier === formObject.autoLoginAuthenticationBackend) {
+          loginUrl = getLoginUrl(option)
+          window.location = loginUrl;
+        }
+      });
```

Comment:
you should be able to use `formObject.loginOptions.find(...)` to get the relevant auth option, instead of the `forEach`. You can the flatten the code a bit and make it more readable like that.

I would also structure this as a guard clause - if the login option is found, end with a `return`, but then move the `ReactDOM.render` out of the `else` again and remove the `else`.

So roughly the code would look like this:

```
if shouldRedirecToAutologin
  loginOption = formObject.loginOptions.find(...)
  if loginOption
    doRedirect
    return

doUsualReactRender
```

This makes it a bit more fail-safe in the sense that the form with login options will still render if someone is messing with query params and the relevant login option is not found.